server/website/website/views.py

```diff
@@ -572,6 +572,7 @@ def handle_result_files(session, files, execution_times=None):
         result.creation_time = now()
         result.observation_start_time = now()
         result.observation_end_time = now()
+        result.next_configuration = {}
```

Comment:
We need to clear the next_configuration of the copied result. Otherwise, the return value of `give_result()` will be confusing.